testing/jobs.merge.cloudbuild.yaml

```diff
@@ -0,0 +1,38 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+steps:
+
+- id: 'Build Container Image'
+  name: 'gcr.io/cloud-builders/gcloud:$_CLOUDSDK_VERSION'
+  dir: '${_SAMPLE_DIR}'
+  args: ['builds', 'submit', '--pack', 'image=gcr.io/${PROJECT_ID}/${_IMAGE}:latest']
+
+- id: 'Pull Container Image'
+  name: 'gcr.io/cloud-builders/docker:latest'
+  args: ['pull', 'gcr.io/${PROJECT_ID}/${_IMAGE}:latest']
+
+- id: 'Tag Container Image'
```

Comment:
what is the need for this extra tagging? (maybe explain in a brief comment, or the readme)

Reply:
We're maintaining this image at `gcr.io/${PROJECT_ID}/${_IMAGE}` and `${_LOCATION}-docker.pkg.dev/${PROJECT_ID}/${_REPO}/${_IMAGE}:latest`, both Container Registry and Artifact Registry. The latter is our recommend practiced but more unwieldy URLs.

Should we introduce a new gcr URL now?

Further aside: we may want to think about regional image hosting so customers on different continents have reduced latency on quickstart deploy.

Reply:
I will update the step names with this info.
What do you mean by a new GCR url? 

Currently, the service image is hosted only in US, but we definitely can update to support more regions

Reply:
AR can [take over gcr.io repository urls](https://cloud.google.com/artifact-registry/docs/transition/setup-gcr-repo) - not a blocker here, but it would simplify any image publishing we do, so i'd recommend we do it at some point.